core/src/main/java/org/mskcc/cbio/portal/scripts/ImportFusionData.java

```diff
@@ -120,6 +121,12 @@ public void importData() throws IOException, DaoException {
                     ProgressMonitor.logWarning("Gene not found:  " + geneSymbol + " ["
                             + entrezGeneId + "]. Ignoring it "
                             + "and all fusion data associated with it!");
+                    continue;
+                }
+                if (genePanel != null && !GeneticProfileUtil.geneInPanel(gene, genePanel)) {
+                    ProgressMonitor.logWarning("Gene is not profiled in gene panel:  " + geneSymbol + " ["
```

Comment:
@n1zea144 any such new warnings in the loader also need to be implemented in the validator. Can you please add it there as well?